web/app/themes/ppj/partials/headerNavigationMenu.php

```diff
@@ -1,40 +1,52 @@
 <?php
-$navMenuItems = wp_get_nav_menu_items( ppj\getLegNameFromPath() );
+global $post;
 
-$logoTargetUrl = ppj\getLogoTargetURL();
+$modifiedSiteWideNavItems = ppj\markCurrentlySelectedAncestorMenuItem(wp_get_nav_menu_items('site-wide-nav'));
+$logoTargetUrl = ppj\getLegHomeRelativePath();
 
-$filteredNavMenuItems = [];
-if (isset($navMenuItems) && !!$navMenuItems ) {
-    foreach ( $navMenuItems as $item ) {
-        $same = ( isset( $post ) && $post->ID == $item->object_id );
-
-        $filteredNavMenuItems[] = [
-            'title'    => $item->title,
-            'url'      => $item->url,
-            'selected' => $same
-        ];
-    }
-}
 ?>
 
 <div class="l-site-header">
     <div class="header__site-header">
-        <div class="header__site-header-content">
+        <div class="header__site-wide-nav">
+            <ul class="header__site-wide-nav-menu-list">
+                <?php foreach ($modifiedSiteWideNavItems as $item):
+                    $listElementClass = 'header__site-wide-nav-menu-list-element';
+                    $listElementClass .= ($item['selected']) ? ' header__site-wide-nav-menu-list-element--selected' : '';
+                    ?>
+                    <li class="<?= $listElementClass ?>">
+                        <a href="<?= $item['url'] ?>"
+                           class="header__site-wide-nav-menu-list-element-link">
+                            <span class="header__site-wide-nav-menu-list-element-text"><?= $item['title'] ?></span>
+                        </a>
+                        <span class="header__nav-link js" onclick="ppj.openNavMenu()">menu</span>
```

Comment:
convert span to button - a button is more semantically correct and can be utilised by a screen reader more readily